File: 2019/Day 01/aoc01_2.py
file = open("aoc01_input.txt")
l = [int(j) for j in file.read().split("\n")]
file.close()

s = 0
for i in l:
	c = i
	while c > 0:
		c = c // 3 - 2

		if c > 0:
			s += c

print(s)

File: 2021/Day 01/aoc01_2.cpp
#include <iostream>
#include <fstream>
using namespace std;

int main() {
	ifstream infile;
	infile.open("aoc01_input.txt");

	int c = 0;
	int last[3] {0, 0, 0};
	int next;
	infile >> last[0];
	infile >> last[1];
	infile >> last[2];
	
	while(infile >> next) {
		if(last[0] < next)
			c++;
		last[0] = last[1];
		last[1] = last[2];
		last[2] = next;
	}
	cout << c;

	return 0;
}